contracts/test/integration/L2GatewayDiamond.t.sol

```diff
@@ -45,11 +47,11 @@ contract L2GatewayActorDiamondTest is Test, L2GatewayActorDiamond {
         IpcEnvelope[] memory topDownMsgs = new IpcEnvelope[](n);
         for (uint64 i = 0; i < n; i++) {
             topDownMsgs[i] = TestUtils.newXnetCallMsg(
-                IPCAddress({subnetId: id, rawAddress: FvmAddressHelper.from(address(this))}),
                 IPCAddress({
                     subnetId: gatewayDiamond.getter().getNetworkName().getParentSubnet(),
                     rawAddress: FvmAddressHelper.from(receipient)
                 }),
+                IPCAddress({subnetId: id, rawAddress: FvmAddressHelper.from(address(this))}),
```

Comment:
Because it was labeled as a top-down message, but the path (from, to) made it a bottom-up message, the test was incorrect. Since we now have checks ensuring that top-down messages are indeed top-down (as a parent can't send a message to itself), this would cause a revert.